roles/lxc/tasks/main.yml

```diff
@@ -1,7 +1,14 @@
 ---
+- name: Set rc.local fact
+  set_fact:
+    rc_local: "{{ lookup('template', 'templates/rc.local.j2') }}"
```

Comment:
any reason we cant just use this directly in line 10 for the content, rather than setting a variable?

If we need to keep this variable, think we should name it something more specific to the role, setting a fact that is just called `rc_local` seems a bit risky if something else needs to play with rc.local for some reason.  Maybe `proxmox_lxc_rc_local` to keep in line with the existing variables?

Applies to the cleanup task as well.

Reply:
I thought about this, but did not test it. I will validate if we even need to set the fact or not no my next test run of this.